crates/tauri/src/menu/plugin.rs

```diff
@@ -824,12 +824,13 @@ fn set_as_help_menu_for_nsapp<R: Runtime>(
   webview: Webview<R>,
   rid: ResourceId,
 ) -> crate::Result<()> {
-  #[cfg(target_os = "macos")]
-  {
-    let resources_table = webview.resources_table();
-    let submenu = resources_table.get::<Submenu<R>>(rid)?;
-    submenu.set_as_help_menu_for_nsapp()?;
-  }
+  // Broken: https://github.com/tauri-apps/muda/issues/263
+  // #[cfg(target_os = "macos")]
+  // {
+  //   let resources_table = webview.resources_table();
+  //   let submenu = resources_table.get::<Submenu<R>>(rid)?;
+  //   submenu.set_as_help_menu_for_nsapp()?;
```

Comment:
we're also calling it incorrectly here: https://github.com/tauri-apps/tauri/blob/4188ffdafcaa7c3c8b000973146c9384a873e741/crates/tauri/src/menu/plugin.rs#L814